events/2021-kubevirt-summit/proposals/vm-snapshots.md

```diff
@@ -0,0 +1,23 @@
+# Title
+
+## Virtual Machine Snapshots
+### Using the KubeVirt snapshot API to back up and restore your VMs
+
+# Abstract
+
+The Virtual Machine Snapshot API was added to KubeVirt in release 0.34.0.  This session discusses the API, it's current implementation, and what is planned for the future.
+
+# Presenters
+
+- Michael Henriksen (henriksen.mike@gmail.com, github: mhenriks)
+- Ryan Hallisey (rhallisey@nvidia.com, github: rthallisey, twitter: @rthallisey)
+
+[X] The presenters agree to abide by the
+    [Linux Foundation's Code of Conduct for Events](https://events.linuxfoundation.org/about/code-of-conduct/)
+
+# Session details
+
+- Track: User
```

Comment:
One question here, based on content of some recent community calls.

Certainly I believe there is value in a session about  snapshots in the user track, but I'm wondering if you might want another for the contributors track? In case more developer-oriented discussions.

Reply:
I'll make this the developer session and I'll create a new proposal for the user session.